client/my-sites/email/email-management/style.scss

```diff
@@ -149,6 +149,22 @@
 .email-plan-mailboxes-list__mailbox-list {
 	margin-top: 15px;
 }
+.email-plan-mailboxes-list__mailbox-action-menu-popover {
+	.email-plan-mailboxes-list__mailbox-action-menu-item {
```

Comment:
Nesting these selectors is probably not necessary. The long BEM-style names should make them unique enough.

Reply:
This was needed to override some of the inbuilt styles in MenuPopover components. I needed to hide the external link icon while preserving its behaviour as an external link to adhere to the given designs.

Reply:
In that case I would do a different selector, like:
```css
.native__menu-item.my-specific-ui__menu-item
```
That has the same specificity and makes it more clear that we are overriding a style and which one it is.

Reply:
Right, makes more sense!